src/cleed++/LEED/LEED.hh

```
@@ -0,0 +1,40 @@
+ *
+ * @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+ *
+ *********************************************************************/
+
+/*! \file
+ *
+ * C++ wrapper class for leed structure.
+ */
+
+#ifndef __LEED_HH__
+#define __LEED_HH__
```

Clang (14.0.6)

[warning] declaration uses identifier '__LEED_HH__', which is a reserved identifier

(bugprone-reserved-identifier)